packages/core/src/internal/verifyTransaction/verifyTransaction.ts

```diff
@@ -34,8 +35,14 @@ export function verifyTransaction(method: string, parameters: any, response: any
         // Transaction already signed, nothing to do
         return;
     }
+
```

Comment:
This is not needed because the broadcasted transaction shall be already signed and will be catch in previous statement. If the broadcastTransaction is not signed, an error will be returned from node anyway.